Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -189,14 +189,17 @@ public final class ProjectPamphletViewModel: ProjectPamphletViewModelType, Proje
       }
       .take(first: 1)
 
-    Signal.combineLatest(
-      freshProjectAndRefTag,
-      cookieRefTag,
+    Signal.zip(
```

Comment:
Note that we'll need to change this to a `combineLatest` once https://github.com/kickstarter/ios-oss/pull/996 merges, because the `fetchProject` request will no longer fire on `viewDidAppear`.